src/FixedDataTableCellGroup.js

```diff
@@ -164,44 +161,12 @@ class FixedDataTableCellGroup extends React.Component {
       this._staticCells[i] = this._renderCell(i, columnIndex);
     }
 
-    var style = {
-      height: props.cellGroupWrapperHeight || props.height,
-      position: 'absolute',
-      width: props.contentWidth,
-      zIndex: props.zIndex,
-    };
-
-    // // // // console.log(this._initialRender,"Hello")
-    FixedDataTableTranslateDOMPosition(
-      style,
-      -1 * props.left,
-      0,
-      this._initialRender,
-      this.isRTL
-    );
-
-    if (this.props.isRTL) {
-      style.right = props.offsetLeft;
-    } else {
-      style.left = props.offsetLeft;
-    }
-
```

Comment:
As we have provided all the styling to a particular cell .So, now there is no need to provide styles to whole cell group .